### PR TITLE
Ability to configure the temporal web ui with helm values

### PR DESCRIPTION
**Pull request by arnobroekhof**
## What was changed
Added the ability to configure the temporal web ui's config.yml by using helm values.

## Why?
because i wanted to configure the web ui by altering values.yml instead of altering the helm-chart directly

2. How was this tested:

* deployed the helm chart to my own kubernetes cluster
* deploy the helm chart to a kubernetes cluster and check the config.yml inside the temporal-web pod

3. Any docs updates needed?
the section https://github.com/temporalio/helm-charts#install-and-configure-temporal can be updated by including web ui config

```
helm install temporaltest \
   -f values/values.cassandra.yaml \
   -f values/values.elasticsearch.yaml \
   --set grafana.enabled=false \
   --set prometheus.enabled=false \
   --set server.replicaCount=5 \
   --set server.config.persistence.default.cassandra.hosts=cassandra.data.host.example \
   --set server.config.persistence.default.cassandra.user=cassandra_user \
   --set server.config.persistence.default.cassandra.password=cassandra_user_password \
   --set server.config.persistence.default.cassandra.tls.caData=$(base64 --wrap=0 cassandra.ca.pem) \
   --set server.config.persistence.default.cassandra.tls.enabled=true \
   --set server.config.persistence.default.cassandra.replicationFactor=3 \
   --set server.config.persistence.default.cassandra.keyspace=temporal \
   --set server.config.persistence.visibility.cassandra.hosts=cassandra.vis.host.example \
   --set server.config.persistence.visibility.cassandra.user=cassandra_user_vis \
   --set server.config.persistence.visibility.cassandra.password=cassandra_user_vis_password \
   --set server.config.persistence.visibility.cassandra.tls.caData=$(base64 --wrap=0 cassandra.ca.pem) \
   --set server.config.persistence.visibility.cassandra.tls.enabled=true \
   --set server.config.persistence.visibility.cassandra.replicationFactor=3 \
   --set server.config.persistence.visibility.cassandra.keyspace=temporal_visibility \
  **--set web.config.auth.enabled=false**
   . \
   --timeout 15m \
   --wait

```

### CODE REVIEW

**Comment by sw-yx** [comment]
i like it! recommend adding a mention to the readme before merging.

**Comment by sw-yx** [comment]
nice :) i will avoid approving as i dont own the helm chart but thank you very much for the contribution. cc @mastermanu

File: templates/web-deployment.yaml [hunk 1]
```diff
@@ -29,14 +29,21 @@ spec:
...
           image: "{{ .Values.web.image.repository }}:{{ .Values.web.image.tag }}"
           imagePullPolicy: {{ .Values.web.image.pullPolicy }}
           env:
             - name: TEMPORAL_GRPC_ENDPOINT
               value: "{{ include "temporal.fullname" . }}-frontend.{{ .Release.Namespace }}.svc:{{ .Values.server.frontend.service.port }}"
-
+          volumeMounts:
+          - name: {{ .Chart.Name }}-web-config
+            mountPath: /usr/app/server/config.yml
+            subPath: config.yml
```
Comment (notofir) [inline templates/web-deployment.yaml:46]:
```suggestion
          volumeMounts:
            - name: {{ .Chart.Name }}-web-config
              mountPath: /usr/app/server/config.yml
              subPath: config.yml
```

**Review by arnobroekhof — COMMENTED** [review]
Fixed indentation

**Comment by arnobroekhof** [comment]
is this still a valid pull request or should i remove the pull request?

**Review by underrun — APPROVED** [review]

**Pull request merged**